.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -62,35 +86,23 @@ You can use `fastmod` to interactively find and replace versions.
 
 For example, you can do something like:
 ```
-fastmod --extensions rs,toml,md --fixed-strings '1.0.0-alpha.12' '1.0.0-alpha.13'
-fastmod --extensions rs,toml,md --fixed-strings '0.2.9' '0.2.10' tower-batch
-fastmod --extensions rs,toml,md --fixed-strings '0.2.8' '0.2.9' tower-fallback
+fastmod --extensions rs,toml,md --fixed-strings '1.0.0-beta.11' '1.0.0-beta.12'
+fastmod --extensions rs,toml,md --fixed-strings '0.2.26' '0.2.27' tower-batch tower-fallback
```

Comment:
` tower-batch` and `tower-fallback` has different versions.

https://github.com/ZcashFoundation/zebra/blob/main/tower-batch/Cargo.toml#L3
https://github.com/ZcashFoundation/zebra/blob/main/tower-fallback/Cargo.toml#L3

Reply:
I gave them the same version in the latest release, to simplify the release process.